packages/app/src/cli/models/extensions/specifications/marketing_activity_schemas/marketing_activity_schema.test.ts

```diff
@@ -139,21 +102,14 @@ describe('MarketingActivityExtensionSchema', () => {
           ...config,
           fields: [
             {
-              ...config.fields[0],
-              min_length: false,
+              key: 'test_field',
+              name: 'test field',
+              description: 'test description',
+              min_length: true,
             },
           ],
         }),
-      ).toThrowError(
-        new zod.ZodError([
-          {
-            message:
-              'Error found on Field "test_field": [\n  {\n    "code": "invalid_type",\n    "expected": "number",\n    "received": "boolean",\n    "path": [\n      "min_length"\n    ],\n    "message": "Expected number, received boolean"\n  }\n]',
-            code: zod.ZodIssueCode.custom,
-            path: ['fields', 0],
-          },
-        ]),
-      )
+      ).toThrow('Field must have a ui_type')
```

Comment:
This looks completely different from the previous test, but was there a behavioral change?